### ISSUE TITLE
Import tracks from a REST API

### ISSUE THREAD

**Issue by carlosribas** [igvteam/igv.js#1611]
Hello,

I would like to create a service to fetch data for a track and use it as described in the first example on this [page](https://github.com/igvteam/igv.js/blob/a86e7aadd70636e25866c32ae01fa0eb1fdb46b5/examples/custom-webservice.html). If my endpoint returns a json with the fields named as a bed file, will IGV be able to import it? 

Also, I can see there is an object called "mapping", but I'm not sure what parameters that object accepts. Where can I find this info?

Thanks!

**Comment by jrobinso** [comment]
That probably means that you apge could not access our server of predefined genomes.   If you aren't using predefined genomes you don't need to worry about it.   I'll see if I can reproduce the issue and print a better warning message.

RE your service, is it working?     The URL is not a pattern I've used before,  it had not occurred to me to try a back-quoted string.   In fact when this bit of the code was developed back-quoted strings were not widely supported in browsers.   The pattern in the example you referenced is what is supported.

**Comment by jrobinso** [comment]
Acutally there was a problem with our genome server.   When this error occurs it should automatically switch to a backup server,  but in any even the primary server has been fixed.

**Comment by carlosribas** [comment]
Yeah, the error is gone. Thank you for your attention to this matter!
